azurelinuxagent/common/cgroupconfigurator.py

```diff
@@ -290,6 +305,12 @@ def __setup_azure_slice():
             if not os.path.exists(vmextensions_slice):
                 files_to_create.append((vmextensions_slice, _VMEXTENSIONS_SLICE_CONTENTS))
 
+            if not os.path.exists(logcollector_slice):
+                from azurelinuxagent.ga.collect_logs import CollectLogsHandler
```

Comment:
can we move this import to the top of the file?

Reply:
I was worried about a cyclical import (as we import `cgroupconfigurator` into `ga.collect_logs`); I haven't actually tried it yet, though, so I can do so and update here.

Reply:
Update: this indeed does cause a cyclical import, so I'm inclined to leave it as is.

Reply:
OK, but coming to think about it, the CGroupConfigurator should not have a dependency on the CollectLogsHandler.

Seems like you just need the value of a few constants? Maybe move those to this module, or move them to a module that both CGroupConfigurator and CollectLogsHandler depend on?